.stylelintrc

```diff
@@ -0,0 +1,135 @@
+{
+  "extends": "stylelint-config-standard",
+  "rules": {
+    "color-named": "never",
+    "declaration-no-important": true,
+    "font-family-name-quotes": "always-where-recommended",
+    "font-weight-notation": "numeric",
+    "function-url-quotes": "always",
+    "selector-max-specificity": "0,2,0",
+    "selector-no-id": true,
+    "selector-root-no-composition": true,
+    "string-quotes": "single",
+    "declaration-block-properties-order": [
```

Comment:
Yes, everything is arguable. Here are my pros:
* Not all colors have human readable format, so we'll definitely have both hex and human readable values at the same time. So we'll end up with color-inconsistency;
* When you edit images or design markup in photo editor - Gimp/Photoshop - you can easily copy color to/from there. You don't need to think what `magenta` or `teal` is; Or when you use color picker to grab color from another place, it always gives you hex or rgb - how could I know if it has human readable alternative?
* Looks like we have only 17 color names that are standartized [1], but what about the rest 123 from [2]? Does it mean that different browsers can choose whatever hex code for `CornflowerBlue` they want? This will be quite nasty bug if it's the case;
* Don't consider this one as pros, it's just about my personal feeling :) So for me using numbers everywhere gives me 100% confidence on the exact value of font-weight/color/border-width we're talking about.
So what are your good points of human readable colors?

[1] https://www.w3.org/TR/CSS22/syndata.html#value-def-color
[2] http://www.w3schools.com/colors/colors_names.asp